selftests/functional/plugin/spawners/test_podman.py

```diff
@@ -0,0 +1,43 @@
+import os
+import shutil
+import unittest
+
+from avocado.utils import process, script
+from selftests.utils import AVOCADO, TestCaseTmpDir
+
+TEST_INSTRUMENTED_PASS = """from avocado import Test
+
+
+class PassTest(Test):
+
+    def test(self):
+        pass
+"""
+
+
+@unittest.skipIf(shutil.which('podman') is None,
+                 "Podman not installed (command podman is missing)")
+class PodmanSpawnerTest(TestCaseTmpDir):
+
+    def test_avocado_instrumented(self):
+        test = script.Script(os.path.join(self.tmpdir.name, "passtest.py"),
+                             TEST_INSTRUMENTED_PASS)
+        test.save()
+        result = process.run(f"{AVOCADO} run "
```

Comment:
A command execution failure here will throw a test `ERROR`.  You may want to properly characterize it as test `FAIL`ure instead (decorators or ignoring the status and failing the test).

Reply:
ok, I will do that